File: eFromAPI/eFormAPI/eFormAPI.BasePn/Models/SearchableList/AdvEntitySearchableGroupEditModel.cs
using System.Collections.Generic;
using eFormData;

namespace eFormAPI.BasePn.Models.SearchableList
{
    public class AdvEntitySearchableGroupEditModel
    {
        public string Name { get; set; }
        public string GroupUid { get; set; }
        public List<EntityItem> AdvEntitySearchableItemModels { get; set; }
    }
}

File: eform-client/src/app/common/models/advanced/searchable-groups/adv-entity-searchable-group-edit.model.ts
import { AdvEntitySearchableItemModel } from 'src/app/common/models';

export class AdvEntitySearchableGroupEditModel {
  name: string;
  advEntitySearchableItemModels: Array<AdvEntitySearchableItemModel> = [];
  groupUid: number;
  isLocked: boolean;
  isEditable: boolean;
  description: string;
}
